### PR TITLE
[RFR] Ajout de correctif (http, error 415, authorization) SPARQL - TEXT

### PR DESCRIPTION
**Pull request by chaBiselx**

### CODE REVIEW

File: src/app/js/user/index.js [hunk 1]
```diff
@@ -57,18 +57,40 @@ export const getRequest = createSelector(
...
+            head['Authorization'] = `Bearer ${token}`;
+        }
+        if (typeof body == 'object') {
+            body = JSON.stringify(body);
+        }
+
+        return {
+            url,
+            body,
+            credentials: cred,
```
Comment (parmentf) [inline src/app/js/user/index.js]: Why did you rename `credentials` parameter to `cred`, and `headers` to `head` (apart from having same length parameters)?
It would have been neat to return 

```js
{
    url,
    body,
    credentials,
    headers,
    method,
}
```

**Review by parmentf — DISMISSED** [review]

File: src/app/js/formats/sparql/SparqlRequest.js [hunk 2]
```diff
@@ -144,6 +145,29 @@ export default url => FormatView => {
...
+                hostname: source.hostname,
+                slashes: source.slashes,
+                port: source.port,
+                pathname: source.pathname,
+                search: source.search,
+            };
+            const url = URL.format(target);
+            return (
+                <p>
+                    {polyglot.t('sparql_http_retry')}
```
[the remark this answers is not in the document]
Comment (chaBiselx) [reply src/app/js/formats/sparql/SparqlRequest.js:165]: done
